WMIAdventure/frontend/src/js/utils/colors.js

```diff
@@ -6,6 +6,7 @@ const colors = {
 
     whiteAlmost: '#FDFDFD',
     light2: '#F7F7F7',
+    light3: '#F2F2F2',
```

Comment:
Do czego ten kolor? Nie mamy takiego na Figmie

Reply:
Bo Michał dodał taki kolor w figmie, ale nie dodał stylu w figmie

Reply:
A możemy ten kolor zmienić na taki który już mamy? Wygląda jakby to było przez pomyłkę

Reply:
a to dawno temu dałem bo mi potrzebny był, lepiej to wywalić i zmienić na inny